names/names.py

```diff
@@ -11,10 +13,51 @@
 f.close()
 
 duplicates = []
+# for name_1 in names_1:
+#     for name_2 in names_2:
+#         if name_1 == name_2:
+#             duplicates.append(name_1)   <= Time Complexity O(n^2) because of double loop
+
+
+# HASHTABLE solution
+# hashtable = {}
+
+# for name_1 in names_1:
+#   hashtable[name_1] = 0
+
+# for name_2 in names_2:
```

Comment:
I realllly really really like the fact that you you took an iterative approach to getting to a better time complexity. You started with o(n^2), got to n(logn) and finally o(n). SUPERB